src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -149,6 +149,8 @@ public static class Path
                 public static readonly string SourceRootMappingDirectory = "SourceRootMapping";
                 public static readonly string SourcesDirectory = "s";
                 public static readonly string TestResultsDirectory = "TestResults";
+                public static readonly string TestDirectory = "Test";
```

Comment:
normally, we don't add code into common lib if they only used by Test, you can just use hard code string in the Test proj. :)